src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs

```diff
@@ -22,7 +22,7 @@ private static bool GetGlobalizationInvariantMode()
                 }
                 else
                 {
-                    int loaded = Interop.Globalization.LoadICU();
+                    int loaded = LoadICUCore();
```

Comment:
```suggestion
                    int loaded = LoadICU();
```
There is no LoadICU method in this type, so the Core suffix is not necessary.